cl/alerts/tasks.py

```diff
@@ -162,34 +165,45 @@ def send_alert_and_webhook(
     d_pk: int,
     since: datetime,
     recap_email_recipients: list[str] = None,
+    docket_entries: list[DocketEntry] = None,
 ) -> None:
     """Send an alert and webhook for a given docket
 
     :param d_pk: The docket PK that was modified
     :param since: If we run alerts, notify users about items *since* this time.
     :param recap_email_recipients: The recap.email addresses if needed to send
     the first case-user notification email.
+    :param docket_entries: A list of docket entries used if we need to send an
+    alert again for a recap.email user independently
```

Comment:
I'm not sure I understand this. Can you say *when* this would be the case?

Reply:
Yes of course, I've explained in detail in the previous comment related to `recap_email_user_only` when this is necessary.
https://github.com/freelawproject/courtlistener/pull/2257#discussion_r988404198